packages/components/src/LegacyPagination/LegacyPagination.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from 'styled-components';
+import { MaterialPlayArrow } from '@t3n/icons';
+import { color, space, typography } from 'styled-system';
+import { Box } from '../Box';
+import { Icon } from '../Icon';
+
+export interface LegacyPaginationProps {
+  page: number;
+  pages: number;
```

Comment:
Vllt. lieber maxPages oder so nennen? Dann ist der Unterschied etwas klarer

Reply:
So hiess es vorher, ich hatte dann pages vorgeschlagen, da ich es in Zusammenhang mit `page` sauberer fand durch den type `number`. `maxPages` find ich irgendwie irreführend, wenn dann müsste es ja `totalPages` oder so sein.

Reply:
mit totalPages bin ich auch fein. Pages suggeriert vom namen her halt ein array mit pages, nicht die max anzahl an pages. Also ich habe es gelesen und mich direkt gefragt warum da ein array mit pages rein muss.

Vielleicht?
`currentPage`, `maxPage`, `maxPageLinks`

Reply:
`currentPage` hatte ich auch überlegt. Find ich gut. + `totalPages`.